linked_list/linked_list.py

```diff
@@ -9,71 +9,142 @@ def __init__(self, value, next_node = None):
 # Defines the singly linked list
 class LinkedList:
     def __init__(self):
-      self.head = None # keep the head private. Not accessible outside this class
+        self.head = None # keep the head private. Not accessible outside this class
 
     # returns the value in the first node
     # returns None if the list is empty
-    # Time Complexity: ?
-    # Space Complexity: ?
+    # Time Complexity: O(N)
+    # Space Complexity: ? O(1)
     def get_first(self):
-        pass
+        current = self.head
+        while current != None:

```

Comment:
We usually prefer to use `is` and `not` when comparing values to what we call "singletons" like `True`, `False`, and `None` the reason being it's slightly more reliable. Occasionally `==` will be overwritten with the`__eq__` method and lead to unexpected behavior.

Also you just want to check `if` the value isn't none, there's no need for iteration so an if statement is the better choice here! 

```suggestion
        if current is not None:

```